lib/trino-orc/src/main/java/io/trino/orc/reader/ColumnReaders.java

```diff
@@ -65,9 +64,6 @@ public static ColumnReader createColumnReader(
             case BOOLEAN:
                 return new BooleanColumnReader(type, column, memoryContext.newLocalMemoryContext(ColumnReaders.class.getSimpleName()));
             case BYTE:
-                if (type == INTEGER && !column.getAttributes().containsKey("iceberg.id")) {
-                    throw invalidStreamType(column, type);
```

Comment:
> Don't require iceberg.id attribute in ORC reader

If I am understanding the change correctly, it could be titled "Fix reading ORC files after column evolved from tinyint to integer"

Am i reading this right?

Reply:
Sorry for my misleading commit tile. Updated it.

Reply:
Thanks for changing the commit title.

Is TestHiveTransactionalTable.java the only test coverage we have for this "Fix reading ORC files after column evolved from tinyint to integer" change?